src/main/java/uz/bookstore/bookstore/dto/response/BookDto.java

```diff
@@ -0,0 +1,59 @@
+package uz.bookstore.bookstore.dto.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import uz.bookstore.bookstore.entity.Book;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Getter
```

Comment:
We do not need @Setter, @NoArgsConstructor annotations remove them from all of the DTOs where they exists.